contracts/multihop/src/tests/swap.rs

```diff
@@ -178,3 +580,41 @@ fn swap_panics_with_no_operations() {
         Err(Ok(ContractError::OperationsEmpty))
     );
 }
+
+fn generate_minted_tokens<'a, const LEN: usize>(
+    env: &Env,
+    admin: Address,
+    number_of_tokens: i32,
+    different_token_ratio: bool,
+) -> [token_contract::Client<'a>; LEN] {
+    let mut tokens = [token_contract::Client, LEN];
+    for i in 0..number_of_tokens {
+        let token = deploy_token_contract(&env, &admin);
+        tokens[i] = token
+    }
+
+    tokens.sort_by(|a, b| a.address.cmp(&b.address));
+
+    match different_token_ratio {
+        true => {
+            let mut i = 1i128;
+            tokens.iter().for_each(|mut t| {
+                t.mint(&admin, &(i * 10_000_000i128));
+                i += 1;
+            });
+        }
+        false => {
+            tokens.iter().for_each(|t| t.mint(&admin, &10_000_000i128));
+        }
```

Comment:
I specifically didn't mention token generation in my comments regarding making helper functions.
1) Sorting doesn't make sense. Tokens needs to be sorted just before introducing as a parameters for liquidity pools creation.
2) Different token ratio as a boolean doesn't make sense. We want to have a control over ratios.
3) Minting tokens doesn't have anything to do with token ratio. It matters during initial liquidity providing, which is I wrote on mattermost:
> let pool = deploy_and_initialize_lp(token_a, token_a_amount, token_b, token_b_amount, Option<fees>, Option<slippage>)

Because that's where using liquidity and token amounts matters.

Reply:
the highlighted code is gone and replaced with the example from above